### ISSUE TITLE
[2017.11.14] The Shattered Gradients Problem: If resnets are the answer, then what is the question?

### ISSUE THREAD

**Issue by unizard** [unizard/AwesomeArxiv#70]

**Comment by unizard** [comment]
### Related work
- DiracNets: Training Very Deep Neural Networks Without Skip-Connections,
